.ci/check-performance-regression.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+set -e
+
+# max difference tolerance in %
+MAX_DIFFERENCE=10
+# baseline of execution time in second
+EXECUTION_TIME_BASELINE=415.72
+
+# sample project path
+SAMPLE_PROJECT="./.ci/jdk17"
+CONFIG_FILE="./.ci/benchmark-config.xml"
+
+# run a command and time it
+# $TEST_COMMAND: command being timed
+time_command() {
+  # Run the command with time
+  local TEST_COMMAND=$1
+  if [ -z "$TEST_COMMAND" ]; then
+        echo "Missing TEST_COMMAND as an argument."
+        exit 1
+    fi
+  # shellcheck disable=SC2086
+  local TIME_OUTPUT=$(command time -p $TEST_COMMAND 2>&1)
+  # Extract execution time
+  local EXECUTION_TIME=$(echo "$TIME_OUTPUT" | awk '/real/ {print $2}')
+
+  echo "${EXECUTION_TIME}"
+}
+
+# insert all the filters into the config file
+generate_config() {
```

Comment:
No generation is required, just make config as you need and store it in git

Reply:
done